maven-core/src/main/java/org/apache/maven/graph/DefaultGraphBuilder.java

```diff
@@ -284,20 +284,7 @@ public DefaultGraphBuilder( BuildResumptionDataRepository buildResumptionDataRep
             excludedProjects.addAll( getProjectsBySelectors( request, projects, optionalSelectors, false ) );
 
             result = new ArrayList<>( projects );
-
-            for ( MavenProject excludedProject : excludedProjects )
```

Comment:
Wasn't this removed in an another commit recently?

Reply:
I think you mean [this one](https://github.com/apache/maven/pull/458)? Indeed - we'll rebase one on top of the other, depending on the order in which we merge them.

Reply:
OK, makes sense.